onedal/datatypes/_data_conversion.py

```diff
@@ -18,28 +18,43 @@
 
 import numpy as np
 
-from daal4py.sklearn._utils import make2d
 from onedal import _backend, _is_dpc_backend
 
-from ..utils import _is_csr
-
 
 def _apply_and_pass(func, *args, **kwargs):
     if len(args) == 1:
         return func(args[0], **kwargs)
     return tuple(map(lambda arg: func(arg, **kwargs), args))
 
 
-if _is_dpc_backend:
+def convert_one_to_table(arg):
+    return _backend.to_table(np.atleast_2d(arg) if np.isscalar(arg) else arg)
+
 
-    from ..utils._dpep_helpers import dpctl_available, dpnp_available
+def to_table(*args):
+    return _apply_and_pass(convert_one_to_table, *args)
 
-    if dpctl_available:
-        import dpctl.tensor as dpt
 
-    if dpnp_available:
+if _is_dpc_backend:
+
+    try:
         import dpnp
 
+        def _onedal_gpu_table_to_array(table, xp=None):
+            # By default DPNP ndarray created with a copy.
+            # TODO:
+            # investigate why dpnp.array(table, copy=False) doesn't work.
+            # Work around with using dpctl.tensor.asarray.
+            if xp == dpnp:
+                return dpnp.array(dpnp.dpctl.tensor.asarray(table), copy=False)
```

Comment:
I don't know is this dpctl module in dpnp is an official part of API? This may work now, but I am not sure for further versions of DPNP. It worth to add some comments here